tests/inventories/patch/group_vars/seed-hosts.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+openshift_cluster_content:
+- object: single patch test
+  content:
+  - name: initialize resource
+    file: "{{ inventory_dir }}/../../files/multi-files-dir/files/route1.yml"
+  - name: patch resource from file
+    file: "{{ inventory_dir }}/../../files/multi-files-dir/files/route1.yml"
+    params: "{{ inventory_dir }}/../../files/patches/patch1.yml"
+    action: patch
+  - name: patch resource from url
+    file: "{{ inventory_dir }}/../../files/multi-files-dir/files/route1.yml"
+    params: "https://raw.githubusercontent.com/redhat-cop/openshift-applier/support-patch/tests/files/patches/patch1.yml"
```

Comment:
The `support-patch` branch will eventually be cleaned out, so this URL should probably reference `master`, no?

Of course that also has the possibility of breaking, which is the reason why none of our current tests references by URL ... but maybe this is the best that can be done?

Reply:
@oybed yeah, I can point it back to master for the merge, but the tests in this PR will fail. Let me try and see if I can get it parameterized to match the branch in travis.

Reply:
@oybed for now I think this is not worth doing, as molecule needs some refactoring first. For now I have pointed to master.

Reply:
@etsauer ok, cool. As an alternative approach, we could just submit this is a separate PR first to ensure it's available in `master`.

Reply:
Also, due to https://github.com/redhat-cop/openshift-applier/issues/85, I don't think the automated tests will fail because of this.